### ISSUE TITLE
Funds stucked since force closure

### ISSUE THREAD

**Issue by AmIAnIssueOpener** [MutinyWallet/mutiny-web#1289]
### TL;DR
I run into a force closure with Mutiny Wallet and funds are stucked since 5 aug 2024.

### Background and issue
I decided to close the channel that I had with mutiny a couple of weeks ago.
Went to the menu section for channels and clicked to close it. I was returned an exception that says:

>  *Channel closed because of an exception: Unable to come to consensus about closing feerate, remote wants something (431 sat) lower than our min fee (770 sat)*

This is the message that appears in the transaction details of the force closure, along with the Channel ID and the date (5 aug 2024).
Since then, I though that I had to wait and the sats would appear eventually in the onchain wallet section. Indeed, if I go to the overview section where LN, fedimint and OC balances are shown separately, I can see the sats under the OC funds. 
I tried to transfer these sats to a new wallet with an OC transaction, but the issue seems to be that there is no actual sats in my OC wallet and any amount I try to transfer is "higher than the wallet balance".

**Comment by TonyGiorgio** [comment]
Provide logs and we can try to take a look

**Comment by AmIAnIssueOpener** [comment]
Thanks for the support!

Here are the logs I can export from Mutiny app as of today:

[mutiny-logs.txt](https://github.com/user-attachments/files/16782783/mutiny-logs.txt)

**Comment by equal11111** [comment]
Having the same exact issue.  Was this ever resolved? What was the fix?

**Comment by FiliTol** [comment]
> Having the same exact issue.  Was this ever resolved? What was the fix?

No solution yet.

**Comment by AmIAnIssueOpener** [comment]
> Having the same exact issue. Was this ever resolved? What was the fix?

No fix at the moment, no

**Comment by TonyGiorgio** [comment]
Your channel close has not been confirmed yet. You can either wait for it to confirm or accelerate it yourself. And then wait a few days for the sweep transaction once the channel force closure confirms. Nothing else we can do here. 

https://mempool.space/tx/1f6fd32780e14fe896e9aca2b3119f3d9d2740d37940bdebb816297c6614fbee
